common-topics/refund.md

```diff
@@ -27,4 +36,4 @@ This is applicable to:
 * [ePayment API](https://developer.vippsmobilepay.com/docs/APIs/epayment-api)
 * [Recurring API](https://developer.vippsmobilepay.com/docs/APIs/recurring-api)
 
-See [FAQ: Refunds](../faqs/reserve-and-capture-faq.md) for common questions.
+See [Reserve and capture](../faqs/reserve-and-capture-faq.md) for common questions.
```

Comment:
Shouldn't this link to https://developer.vippsmobilepay.com/docs/vipps-developers/faqs/refunds-faq/ instead?